Update Export File Format wrt Media Signing

bjornvolcker opened the pull request:
When exporting a segment of a stream to MP4 there are some things to consider to keep Media Signing working.

Commit:
Update Export File Format wrt Media Signing

When exporting a segment of a stream to MP4 there are some
things to consider to keep Media Signing working.

bsriramprasad commented on doc/ExportFileFormat.xml:
```diff
@@ -251,7 +251,7 @@
       <title>Use case 2: Forensic analysis at court</title>
       <para>A court receives video clips from a grocery store, a street surveillance system and a metro operator. All three videos are shown in the court’s approved video player.</para>
-      <para>The judges want to see the suspect in all three video clips with exact time information. They also want to have information when the video clips have been exported and whether the video sequence is complete and authentic.</para>
+      <para>The judges want to see the suspect in all three video clips with exact time information. They also want to have information when the video clips have been exported and whether the video sequence is complete and authentic. Further, they preferably also want to validate the videos authenticity and provenance using ONVIF Media Signing.</para>
```
```suggestion
      <para>The judges want to see the suspect in all three video clips with exact time information. They also want to have information when the video clips have been exported and whether the video sequence is complete and authentic. Further, they preferably also want to validate the authenticity and provenance of videos using ONVIF Media Signing.</para>
```

bsriramprasad commented on doc/ExportFileFormat.xml:
```diff
@@ -260,6 +260,79 @@
+    <section>
+      <title>Requirements to Preserve Media Signing</title>
+      <para>
+        This checklist captures the essence for preserving the authenticity of ONVIF Media Signing
+        (OMS) when handling streams and exporting to MP4. Following these rules ensures that signature
+        verification remains intact and that tamper‐evident evidence can be trusted.
```
```suggestion
        This checklist captures the essence for preserving the authenticity of ONVIF signed streaming media as specified in ONVIF Media Signing
        (OMS) specification when exporting to MP4. Following these rules ensures that signature
        verification remains intact and video evidence can be trusted as tamper-proof.
```

bjornvolcker committed:
Update doc/ExportFileFormat.xml

Co-authored-by: Sriram Bhetanabottla <sriram.bhetanabottla@axis.com>

bjornvolcker committed:
Update doc/ExportFileFormat.xml

Co-authored-by: Sriram Bhetanabottla <sriram.bhetanabottla@axis.com>

bsriramprasad committed:
paraphrasing as requirements text (#623)

Co-authored-by: Sriram Bhetanabottla <sriram.bhetanabottla@axis.com>
---------
Co-authored-by: Björn Völcker <65532189+bjornvolcker@users.noreply.github.com>

bsriramprasad commented:
> Interesting information for implementers but not written as requirements. As it is more a type of explanation on how to handle FFMPEG I suggest to add it as annex.

Contributed to Björn changes to address feedback.

Commit:
Moved most of the Media Signing text to an Appendix

Kept what is truly a requirement and moved the rest as
recommendations to an Appendix.

HansBusch commented on doc/ExportFileFormat.xml:
```diff
@@ -251,7 +251,7 @@
       <title>Use case 2: Forensic analysis at court</title>
       <para>A court receives video clips from a grocery store, a street surveillance system and a metro operator. All three videos are shown in the court’s approved video player.</para>
-      <para>The judges want to see the suspect in all three video clips with exact time information. They also want to have information when the video clips have been exported and whether the video sequence is complete and authentic.</para>
+      <para>The judges want to see the suspect in all three video clips with exact time information. They also want to have information when the video clips have been exported and whether the video sequence is complete and authentic. Further, they preferably also want to validate the authenticity and provenance of videos using ONVIF Media Signing.</para>
```
Prefer the original sentence as a non-technical description. 

Media-Signing proves authenticity and is covered by the original description.

bjornvolcker replied:
Agree. Will remove

HansBusch committed:
Update according to session on requirements of media signing.

bjornvolcker committed:
Removed white space and replaced OMS

bjornvolcker committed:
One use of "should"

kieran242 reviewed (approved):
@bjornvolcker Approved with updates during zoom call.

ocampana-videotec commented:
@bjornvolcker , @HansBusch , @bsriramprasad and @kieran242 this PR still has the On Hold tag, but to me it looks ready for merge.

Does anyone of you object merging?
